internal/wazeroir/operations.go

```diff
@@ -730,6 +732,15 @@ type Label struct {
 	Kind    LabelKind
 }
 
+// LabelID is the unique identifier for blocks in a single functions.
+type LabelID = uint64
```

Comment:
supernit is this an alias for a reason?

Reply:
no reason :D

Reply:
https://github.com/tetratelabs/wazero/pull/1142/commits/6c345a6ff9f62cc94d486335ccbbdef281fce5d2